editor/src/components/canvas/controls/select-mode/select-mode.spec.browser.ts

```diff
@@ -0,0 +1,338 @@
+import { act, fireEvent } from '@testing-library/react'
+import * as TP from '../../../../core/shared/template-path'
+import { wait } from '../../../../utils/test-utils'
+import { makeTestProjectCodeWithSnippet, renderTestEditorWithCode } from '../../ui-jsx.test-utils'
+import { CanvasControlsContainerID } from '../new-canvas-controls'
+
+describe('Select Mode Selection', () => {
+  beforeAll((done) => {
```

Comment:
It's maybe worth moving the contents of this function to a test utils file

Reply:
I moved it and also fixed the function based on the feedback, plus I found an issue with it –
I used `currentWindow.getSize().width` and it turns out that this is how it's used:
`const [width, height] = currentWindow.getSize()` 
it took me 30 minutes to figure out that it's angle brackets, not curly braces... and the return value is an array not an object 🤦

Reply:
Oh my word...